src/Aspire.Hosting/Dashboard/DashboardServiceHost.cs

```diff
@@ -23,7 +23,7 @@ namespace Aspire.Hosting.Dashboard;
 internal sealed class DashboardServiceHost : IHostedService
 {
     private const string DashboardServiceUrlVariableName = "DOTNET_DASHBOARD_GRPC_ENDPOINT_URL";
-    private const string DashboardServiceUrlDefaultValue = "http://localhost:18999";
+    private const string DashboardServiceUrlDefaultValue = "https://localhost:18999";
```

Comment:
Why is this https?

Reply:
gRPC HTTP/2

Reply:
Should be HTTP still hardcoded to H2 as the protocol (we do this today).

Reply:
> gRPC HTTP/2

Only HTTP/2 happens on this port. Pre-negotiated HTTP/2 (both the client and server are configured to only support HTTP/2) is supported without TLS.

Reply:
Fixed.